### PR TITLE
data: amit0804-tech

### PR DESCRIPTION
**Pull request by amit0804-tech**

### CODE REVIEW

File: amit0804-tech.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "url": "https://www.instagram.com/amit_0804/",
+      "icon": "FaInstagram"
+    },
+      {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/amit0804-tech",
+      "icon": "FaGithub"
+    }, {
+      "name": "Follow me on Discord",
+      "url": "amit0804",
```
Comment (kumarsonsoff3) [inline amit0804-tech.json:21]: This is an invalid URL. Please update it with your profile URL.

File: amit0804-tech.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "icon": "FaDiscord"
+    },{
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/amit-nalawade-28b039205/",
+      "icon": "FaLinkedin"
+    },
```
Comment (sital002) [inline amit0804-tech.json:27]:
```suggestion
    }
```
The comma is not required

Comment (kumarsonsoff3) [inline amit0804-tech.json:27]: Also, remove this trailing comma. 

```suggestion
    }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Your file should be in data directory. Please rename your file here to `data/amit0804-tech.json`. 

Change the `amit0804-tech.json`  ->  `data/amit0804-tech.json`

Also, make the changes suggested below.

**Comment by kumarsonsoff3** [comment]
Closing this PR as two new PRs has already been created with the updated changes! #9282, #9284

**Comment by amit0804-tech** [comment]
Awesome, thanks for the update!

**Pull request closed without merging**